introduction.md

```diff
@@ -1,2 +1,6 @@
 Hello all!
 My favorite Code School path is Ruby.
+*javascript
+*c
+*c++
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?